e2e/app/mocksymbiotic.go

```diff
@@ -0,0 +1,171 @@
+package app
+
+import (
+	"context"
+
+	"github.com/omni-network/omni/contracts/bindings"
+	"github.com/omni-network/omni/e2e/app/eoa"
+	"github.com/omni-network/omni/lib/errors"
+	"github.com/omni-network/omni/lib/log"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+type TokenDeployment struct {
+	WstETH common.Address
+	RETH   common.Address
+	METH   common.Address
+}
+
+type SymbioticVaultDeployment struct {
+	WstETHVault common.Address
+	RETHVault   common.Address
+	METHVault   common.Address
+}
+
+func DeployMockSymbiotic(ctx context.Context, def Definition) (map[uint64]TokenDeployment, map[uint64]SymbioticVaultDeployment, error) {
+	deployedTokens, err := deployTokens(ctx, def)
+	if err != nil {
+		return nil, nil, errors.Wrap(err, "deploy tokens")
+	}
+
+	deployedVaults, err := deploySymbioticVaults(ctx, def, deployedTokens)
+	if err != nil {
+		return nil, nil, errors.Wrap(err, "deploy symbiotic vaults")
+	}
+
+	return deployedTokens, deployedVaults, nil
+}
+
+func deployTokens(ctx context.Context, def Definition) (map[uint64]TokenDeployment, error) {
+	tokens := []struct {
+		name   string
+		symbol string
+	}{
+		{"Wrapped Staked ETH", "wstETH"},
+		{"Rocket Pool ETH", "rETH"},
+		{"Mantle ETH", "mETH"},
+	}
+
+	// Track deployed token addresses per chain
+	deployments := make(map[uint64]TokenDeployment)
+
+	for _, chain := range def.Testnet.EVMChains() {
+		backend, err := def.Backends().Backend(chain.ChainID)
+		if err != nil {
+			return nil, errors.Wrap(err, "backend", "chain", chain.Name)
+		}
+
+		deployer := eoa.MustAddress(def.Testnet.Network, eoa.RoleDeployer)
+		auth, err := backend.BindOpts(ctx, deployer)
+		if err != nil {
+			return nil, errors.Wrap(err, "failed to get TransactionOpts for bindings")
+		}
+
+		var deployment TokenDeployment
+		// Deploy each token
+		for i, token := range tokens {
+			addr, _ /* tx */, mockToken, err := bindings.DeployMockToken(auth, backend, token.name, token.symbol)
+			// receipt, _ := backend.WaitMined(ctx, tx)
+			if err != nil {
+				return nil, errors.Wrap(
+					err,
+					"failed to deploy mock token",
+					"chain", chain.Name,
+					"token", token.symbol,
+				)
+			}
+
+			// Store address in deployment struct
+			switch i {
+			case 0:
+				deployment.WstETH = addr
+			case 1:
+				deployment.RETH = addr
+			case 2:
+				deployment.METH = addr
+			}
+
+			log.Info(ctx, "Token deployed",
```

Comment:
lets make this debug, info should only be main workflow steps, this is a sub-step